### DISCUSSION TITLE
if user use docker-composed build their use case ,is this kindof version belongs to community version? if so, does it enable user turn on the HW-acceleration option?

### DISCUSSION THREAD

**Discussion by afrojewelz** [photoprism/photoprism#2776] [Q&A]
if user use docker-composed build their use case ,is this kindof version belongs to community version? if so, does it enable user turn on the HW-acceleration option? i KINDA know through document if user wanna enable HW-acceleration requires 2 have-to steps:
1st you can map hardware symbolink directory into container like rw mount /dev/dri
2nd you are know which kind of PHOTOPRISM_INIT tag suits your hardware with correct driver and ready their api for call

i wonder whether if there are 3rd condition for self-hosted version ? : you gotta be a patreon member in order to utilize hw-acceleration in photoprism

i had checked patreon price list, and noticed that HW-acceleration like cheap hardware with TPU/QSV option is not available with community version , i am still wondering is self-hosted composed photoprism belongs to community version? 

can self build photoprism at least leave this option for user to decide turn on/off?

btw i am all support those themes / geo-location / mapapi charges some reasonable fees.

**Comment by lastzero** [comment]
As a developer you should be able to enable it anyway. Reason we don't like non sponsors using it is the many support requests it creates and generally the amount of time it takes to maintain this feature. Check how many related GitHub Issues there are...

**Reply by afrojewelz** [reply]
i did checked them a lot, this is frustrating when hardware keeps evolving ,feature we yet to got opportunity utilize just about to absolute ,i can understand developer wont'be able handle such enormous demand for free,might i ask how much price for like lifetime personal membership will be?
